### ISSUE TITLE
Forms link in the backend gives Fatal Error

### ISSUE THREAD

**Issue by waynetheisinger** [Stillat/the-agency-starter-kit#3]
## What was I doing
I clicked on the Forms Link in the left hand navigation of the control panel
## What did I expect to happen
That the Create Forms starter page would display
## What actually happened
https://flareapp.io/share/17WJg63P

**Comment by waynetheisinger** [comment]
**Additional Info:** removing all `__(<value>)` calls in `vendor/statamic/cms/resources/views/forms/index.blade.php` makes the error go away... which made me think that the first thing is to verify is the translation files and ensure that for every key you're using with the __('key') function, the corresponding value in your translation files (located in resources/lang) is a string, not an array. However, there doesn't seem to be a resources/lang folder... `¯\_(ツ)_/¯`

**Comment by waynetheisinger** [comment]
More Info: If you change all instances of `__('Forms')` to `__('Foobar')` in `vendor/statamic/cms/resources/views/forms/index.blade.php` then the error goes away

**Comment by JohnathonKoster** [comment]
Thanks for the extra info! I'll look to refactor some things this weekend; I ran into a similar problem on another project with recent versions of Statamic

**Comment by JohnathonKoster** [comment]
I've updated the starter kit to address this issue; you'll need to replace the starter kit's files in order to resolve it (or create a new project with the starter kit).

The root case was the Starter Kit's `/lang/en/forms.php` translation file, which caused an issue wherever there is `__('Forms')`  (or similar) translations anywhere in the project. I've prefixed these files with `agency_` and updated the translation calls.

Happy coding!
